contribute.md

```diff
@@ -12,6 +12,107 @@ The Foreman has a growing list of community events - check the [events
 page](/events) for more details. If there's nothing suitable in your area, why
 not consider [creating an event](https://github.com/theforeman/theforeman.org/blob/gh-pages/_data/events.yml)!
 
+## Installing from Source
```

Comment:
This should probably be combined with the "Setting up development environment" section below, a lot of it is duplicated

Reply:
Merged.